apis/v1alpha1/tcproute_types.go

```diff
@@ -51,15 +54,19 @@ type TCPRouteStatus struct {
 // TCPRouteRule is the configuration for a given rule.
 type TCPRouteRule struct {
 	// Matches define conditions used for matching the rule against
-	// incoming TCP connections.
-	// Each match is independent, i.e. this rule will be matched
-	// if **any** one of the matches is satisfied.
+	// incoming TCP connections. Each match is independent, i.e. this
+	// rule will be matched if **any** one of the matches is satisfied.
+	// If unspecified, all requests from the associated gateway TCP
+	// listener will match.
 	//
 	// +optional
 	// +kubebuilder:validation:MaxItems=8
 	Matches []TCPRouteMatch `json:"matches,omitempty"`
 
-	// ForwardTo defines the backend(s) where matching requests should be sent.
+	// ForwardTo defines the backend(s) where matching requests should be
+	// sent. If unspecified, matching requests will be dropped and a 503
+	// error code returned.
```

Comment:
Maybe ForwardTo should be required on these routes? Admittedly doesn't make sense where we have filters, but I'm not sure if we'd ever add something like that here. Not sure on this one...

Reply:
I'll make the `ForwardTo` required for all route types other than `HTTPRoute`. We can readdress post v1alpha1.